esbuild.config.mjs

```diff
@@ -22,7 +22,7 @@ const entryPoints = [
 ]
 const watchDirectories = [
   "./app/javascript/**/*.js",
-  "./app/views/**/*.html.erb",
+  "./app/views/**/*.html*.erb",
```

Comment:
Let's just do this for any .erb files. Then it will apply to turbo_stream.erb, etc.

```suggestion
  "./app/views/**/*.erb",
```